plothist/plothist_style.py

```diff
@@ -340,7 +340,7 @@ def add_luminosity(
         text += " "
     if is_data:
         if lumi:
-            text += rf"$\int\,\mathcal{{L}}\,\mathrm{{dt}}={lumi}\,{lumi_unit}^{{-1}}$"
+            text += rf"$\int\,\mathcal{{L}}\,\mathrm{{d\mathit{{t}}}}={lumi}\,{lumi_unit}^{{-1}}$"
```

Comment:
Could you try instead `\mathrm{{d}}\mathit{{t}}`? This should give the same result, but the syntax is more logical.